src/lib/network/network_autoyast.rb

```diff
@@ -22,6 +22,19 @@ def initialize
       Yast.import "Linuxrc"
     end
 
+    # Clears ifcfg files to be ready for proposals done later by the installer
+    #
+    # @return [void]
+    def clear_ifcfgs
```

Comment:
The name and the comment make it sound like the entire contents of the ifcfgs are emptied. Compare http://www.rubydoc.info/search/stdlib/core?q=clear